README.md

```diff
@@ -1,10 +1,43 @@
-# The Carpentries Workbench Template Markdown Lesson
+# Model Coupling with MUSCLE3 and AMUSE
 
-This lesson is a template lesson that uses [The Carpentries Workbench][workbench].
+This lesson is intended for people who would like to combine several computational models into one big computational model. It covers the theory behind model coupling and teaches application via use of two different model coupling frameworks: MUSCLE3 and AMUSE.
 
-To get started using this template, visit https://github.com/carpentries/workbench-template-md/generate
-and follow the instructions.
+[![Create a Slack Account with us](https://img.shields.io/badge/Create_Slack_Account-The_Carpentries-071159.svg)](https://swc-slack-invite.herokuapp.com/)
 
-If you have any questions, contact [@zkamvar](https://github.com/zkamvar)
+This repository generates the corresponding lesson website from [The Carpentries](https://carpentries.org/) repertoire of lessons.
```

Comment:
Maybe this should refer to the eScience Academy or whatever it's called at the moment instead of The Carpentries?

Reply:
Good point... I have just removed that line entirely for now. We can add more specific carpentries stuff to the README when we revisit the course (after the pilot)